localisation/wc_culture_buildings.csv

```diff
@@ -1,212 +1,212 @@
+ca_culture_murloc_1;Murloc Elemental Totems;;Murloc-Elementartotems;;;;;;;;;;;x
+ca_culture_murloc_2;Murloc Elemental Totems;;Murloc-Elementartotems;;;;;;;;;;;x
+ca_culture_murloc_3;Murloc Elemental Totems;;Murloc-Elementartotems;;;;;;;;;;;x
+ca_culture_murloc_4;Murloc Elemental Totems;;Murloc-Elementartotems;;;;;;;;;;;x
+ca_culture_murloc_desc;Though the magic of murlocs is relatively primitive compared to other shamanic cultures, murlockind shows a great aptitude and connection to water and earth elemental magics. The magic may be tame and fairly basic, but is terrifying in power when murloc casters come together in great numbers, supported by their fanatic, spear-swinging, followers.;;Obwohl die Magie der Murlocs im Vergleich zu anderen schamanischen Kulturen relativ primitiv ist, zeigt die Art der Murlocs eine gro�e Begabung und Verbundenheit mit der Elementarmagie des Wassers und der Erde. Die Magie mag zahm und recht simpel wirken, aber ihre Macht ist erschreckend, wenn Murloc-Wirker in gro�er Zahl zusammenkommen, unterst�tzt von ihren fanatischen, speerschwingenden Anh�ngern.;;;;;;;;;;;x
+ca_culture_highborne_1;Highborne Pinnacle;;Hochgebohrenen-Spitze;;;;;;;;;;;x
+ca_culture_highborne_2;Highborne Pinnacle;;Hochgebohrenen-Spitze;;;;;;;;;;;x
+ca_culture_highborne_3;Highborne Pinnacle;;Hochgebohrenen-Spitze;;;;;;;;;;;x
+ca_culture_highborne_4;Highborne Pinnacle;;Hochgebohrenen-Spitze;;;;;;;;;;;x
+ca_culture_highborne_desc;The highborne elves are a dying breed of descendants from Azshara's great empire from long ago. Even though it has been so long, the highborne still find themselves incredibly skilled with arcane magic and enchantments, so much so that highborne society finds everything and everyone infused and empowered by some particle of magical enchantment. Near the center of it all is this enormous pinnacle built to better concentrate the ley energies, what powers highborne society, from the surrounding landscapes.;;Die Hochgeborenen sind eine aussterbende Rasse von Nachkommen aus Azsharas einst gro�em Reich. Obgleich �onen vergangen sind, verf�gen die Hochgeborenen nach wie vor �ber unglaubliche F�higkeiten im Umgang mit arkaner Magie und Verzauberungen, so dass ihre Gesellschaft alles und jeden von irgendeinem Partikel magischer Verzauberung erf�llt vorfindet. Im Zentrum befindet sich eine riesige Spitze, die gebaut wurde, um die Ley-Energien, die die Hochgeborenen-Gesellschaft antreibt, aus den umliegenden Landschaften besser zu konzentrieren.;;;;;;;;;;;x
```

Comment:
Edited first sentence.
```suggestion
ca_culture_highborne_desc;The highborne elves are a dying breed, the remnants of Azshara's great empire from long ago. Even though it has been so long, the highborne still find themselves incredibly skilled with arcane magic and enchantments, so much so that highborne society finds everything and everyone infused and empowered by some particle of magical enchantment. Near the center of it all is this enormous pinnacle built to better concentrate the ley energies, what powers highborne society, from the surrounding landscapes.;;Die Hochgeborenen sind eine aussterbende Rasse von Nachkommen aus Azsharas einst gro�em Reich. Obgleich �onen vergangen sind, verf�gen die Hochgeborenen nach wie vor �ber unglaubliche F�higkeiten im Umgang mit arkaner Magie und Verzauberungen, so dass ihre Gesellschaft alles und jeden von irgendeinem Partikel magischer Verzauberung erf�llt vorfindet. Im Zentrum befindet sich eine riesige Spitze, die gebaut wurde, um die Ley-Energien, die die Hochgeborenen-Gesellschaft antreibt, aus den umliegenden Landschaften besser zu konzentrieren.;;;;;;;;;;;x
```

Reply:
👍